NOTICE.txt

```diff
@@ -36,20 +36,36 @@ Any use or displaying shall constitute an infringement under intellectual proper
 
 img/orange-logo.svg
 dist/img/orange-logo.svg
-site/favicon.ico
```

Comment:
I'm not 100% sure but I'd say that `site/favicon.ico` here in fact refers to `_site/favicon.ico`.
Because `site/docs/**/assets/img/favicons/mstile-150x150.png` doesn't exist either, it can be:
* `_site/docs/5.2/assets/img/favicons/mstile-150x150.png`
* `site/static/docs/5.2/assets/img/favicons/mstile-150x150.png`

So maybe we should mention both URLs?

Reply:
Ah, I see, maybe we should, I'm not hundred percent sure since `_site` isn't on main. Otherwise we should mention all `gh-pages` aswell ?
But all urls should have a `static`, changing it !

Reply:
Let's drop `_site/*` references because they are not actually in the repo and rather focus on the `site/*` references.